.github/workflows/build-cirros-bionic.yaml

```diff
@@ -0,0 +1,189 @@
+name: CirrOS image builder
```

Comment:
rename to build-cirros.yaml so no file renames will be needed when ubuntu release change